src/qrCard/index.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+import { space } from '_utils/branding'
+
+import QrCard from './QrCard'
+
+const StyledQrCard = styled(QrCard)`
+  & {
+    display: block;
+    padding: ${space.xl};
+  }
+
+  & img {
+    width: 100%;
+    padding: ${space.m} ${space.xl};
+  }
+
+  & .kirk-item {
+    padding-top: 0;
```

Comment:
I missed that on my first. We should not touch the vertical padding of kirk Items.
We don't want to couple the rendering of these two items by hacking some internal css: Either use a raw Item or use some custom html/css.
If it's really an Item, please challenge the UX on this project and ask him why it does not use the natural 16px padding top/bottom.

Reply:
Yes it's an item info and I think there is no need to override the padding. The card should have no padding and the image should just have a padding of 16px  top and bottom.

```
& {
    display: block;
    padding: ${space.xl}; // remove
  }
  & img {
    width: 100%;
    padding: ${space.m} ${space.xl}; // ${space.m} ${space.m};
  }
```